js/localization/cs.json

```diff
@@ -0,0 +1,239 @@
+{
+    "cs": {
+        "Yes": "Ano",
+        "No": "Ne",
+        "Cancel": "Zrušit",
+        "Clear": "Smazat",
+        "Done": "Ok",
```

Comment:
Hi, Laupe!
Is it right to use just "Ok" here? It might be confused with another OK constant string here. How about "Hotový"?

Reply:
Done could be "OK" or "Hotovo" or "Dokončit". It depends on context